src/README.rst

```diff
@@ -16,6 +16,10 @@ To get started, after installation run the following:
 Change Log
 ==========
 
+7.0.3
```

Comment:
Label this as "Unreleased"

Reply:
is there a specific format. Just adding (Unreleased) following the 7.0.3 is sufficient ?

Reply:
Sorry, I meant change the 7.0.3 to Unreleased. I'll add in the version before release. When I wrote that comment, I had assumed that this would be released with some batch of changes, potentially with the release of the next service fabric version, at the latest. Is this a change that should go out before that? If not, can you actually remove the version changes from that test file as well? Thanks!